src/rgw/rgw_aio.h

```diff
@@ -60,24 +60,24 @@ class Aio {
  public:
   virtual ~Aio() {}
 
-  virtual ResultList submit(RGWSI_RADOS::Obj& obj,
-                            const rgw_raw_obj& raw_obj,
-                            librados::ObjectReadOperation *op,
-                            bufferlist *data, uint64_t cost) = 0;
+  virtual AioResultList submit(RGWSI_RADOS::Obj& obj,
```

Comment:
now -this- is ugly: RGWSI_RADOS::

Reply:
yeah. the main reason i don't like nested classes is that we have to #include "services/svc_rados.h" here just to define the virtual interface